stor/exceptions.py

```diff
@@ -63,6 +71,22 @@ class UnavailableError(RemoteError):
     pass
 
 
+class ConflictError(RemoteError):
+    """Thrown when a 409 response is returned.
+
+    Notes:
```

Comment:
do you want these notes in the actual exception which inherits from `ConflictError` ?

Reply:
what do you mean?

Reply:
just to move the second part of comment comment to `RestoreAlreadyInProgressError` class

Reply:
looks like you already did though.